clients/identity/authentication.go

```diff
@@ -101,6 +114,33 @@ func (api IdentityClient) CheckRequest(req *http.Request) (context.Context, int,
 	return ctx, http.StatusOK, nil
 }
 
+func splitTokens(florenceToken, authToken string) (logData log.Data) {
+	var ft, at tokenObject
+	if len(florenceToken) > 0 {
+		ft = splitToken(florenceToken)
+	}
+
+	if len(authToken) > 0 {
+		at = splitToken(authToken)
+	}
+
+	return log.Data{"florence_token": ft, "auth_token": at}
```

Comment:
I wonder if its worth only adding to the log.Data if the token is not empty, as only one of the two tokens will be present?